src/views/dashboard/DashboardIndex.vue

```diff
@@ -1,112 +1,391 @@
+								<button class="schema-toggle">
+									<ChevronDown v-if="!expandedSchemas.has(schema.id)" :size="20" />
+									<ChevronUp v-else :size="20" />
+								</button>
 							</div>
-							<span class="schema-title">{{ schema.title }}</span>
+
+							<!-- Schema Statistics Table -->
+							<table v-if="expandedSchemas.has(schema.id)" class="statistics-table schema-stats">
+								<thead>
+									<tr>
+										<th>Type</th>
+										<th>Total</th>
+										<th>Size</th>
+									</tr>
+								</thead>
+								<tbody>
+									<tr>
+										<td>Objects</td>
+										<td>{{ schema.stats?.objects?.total || 0 }}</td>
+										<td>{{ formatBytes(schema.stats?.objects?.size || 0) }}</td>
+									</tr>
+									<tr class="sub-row">
+										<td class="indented">
+											Invalid
+										</td>
+										<td>{{ schema.stats?.objects?.invalid || 0 }}</td>
+										<td>-</td>
+									</tr>
+									<tr class="sub-row">
+										<td class="indented">
+											Deleted
+										</td>
+										<td>{{ schema.stats?.objects?.deleted || 0 }}</td>
+										<td>-</td>
+									</tr>
+									<tr>
+										<td>Logs</td>
+										<td>{{ schema.stats?.logs?.total || 0 }}</td>
+										<td>{{ formatBytes(schema.stats?.logs?.size || 0) }}</td>
+									</tr>
+									<tr>
+										<td>Files</td>
+										<td>{{ schema.stats?.files?.total || 0 }}</td>
+										<td>{{ formatBytes(schema.stats?.files?.size || 0) }}</td>
+									</tr>
+								</tbody>
+							</table>
 						</div>
 					</div>
 				</div>
 			</div>
-			<NcEmptyContent v-else
-				name="No Registers"
-				description="No registers have been created yet">
-				<template #icon>
-					<DatabaseOutline />
-				</template>
-			</NcEmptyContent>
 		</div>
 	</NcAppContent>
 </template>
 
 <script>
-import { NcAppContent, NcEmptyContent } from '@nextcloud/vue'
+import { NcAppContent, NcEmptyContent, NcLoadingIcon, NcActions, NcActionButton } from '@nextcloud/vue'
 import DatabaseOutline from 'vue-material-design-icons/DatabaseOutline.vue'
 import FileCodeOutline from 'vue-material-design-icons/FileCodeOutline.vue'
+import ChevronDown from 'vue-material-design-icons/ChevronDown.vue'
+import ChevronUp from 'vue-material-design-icons/ChevronUp.vue'
+import DotsHorizontal from 'vue-material-design-icons/DotsHorizontal.vue'
+import Pencil from 'vue-material-design-icons/Pencil.vue'
+import TrashCanOutline from 'vue-material-design-icons/TrashCanOutline.vue'
+import Upload from 'vue-material-design-icons/Upload.vue'
+import Export from 'vue-material-design-icons/Export.vue'
+import ApiIcon from 'vue-material-design-icons/Api.vue'
+import Download from 'vue-material-design-icons/Download.vue'
+import Calculator from 'vue-material-design-icons/Calculator.vue'
+import { ref, computed, onMounted } from 'vue'
+import { useDashboardStore } from '../../store/modules/dashboard.js'
+import { useNavigationStore } from '../../store/modules/navigation.js'
```

Comment:
For code consistency we always set stores on top of the file as script setup

~~~
<script setup>
import { useDashboardStore } from '../../store/modules/dashboard.js'
import { useNavigationStore } from '../../store/modules/navigation.js'
</script>
~~~

Reply:
Changed